### PR TITLE
  Update image to use rel/1.0.0 builds 

### PR DESCRIPTION
**Pull request by naamunds**
The rel/1.0.0 branch represents the next release of the .NET CLI.

Fixes https://github.com/dotnet/dotnet-docker-nightly/issues/22.  Also related to https://github.com/dotnet/dotnet-docker-nightly/issues/14 (rename from "preview" to "nightly").

cc: @MichaelSimons @dleeapho

### CODE REVIEW

**Comment by naamunds** [comment]
Unfortunately it looks like the history for the README.md can't be preserved when viewed in GitHub after the move from the `preview` directory to the `rel-1.0.0` directory.  The history can still be seen from the command line by using `git log --follow rel-1.0.0/README.md`, but it looks like GitHub still doesn't support tracking that history even though it's been requested for a while (see http://stackoverflow.com/questions/5646174/github-follow-history-by-default).

You can see the diff of the file by viewing https://github.com/dotnet/dotnet-docker-nightly/commit/4246659a3f8f30d0a940a1da0653b490249cba67.

File: rel-1.0.0/Dockerfile [hunk 1]
```diff
@@ -22,8 +22,8 @@ RUN apt-get update \
         zlib1g \
     && rm -rf /var/lib/apt/lists/*
 
-# Install the LKG preview build of the .NET CLI
-RUN wget -q https://dotnetcli.blob.core.windows.net/dotnet/preview/Binaries/Latest/dotnet-dev-debian-x64.latest.tar.gz \
+# Install the LKG rel/1.0.0 build of the .NET CLI
```
Comment (MichaelSimons) [inline rel-1.0.0/Dockerfile]: What do you think of removing this from the comment?  IMO it doesn't add value and is a maintenance issue.

**Comment by MichaelSimons** [comment]
:shipit:

File: rel-1.0.0/Dockerfile (hunk 1 above)
Comment (naamunds) [inline rel-1.0.0/Dockerfile]: Makes sense.  I've made that change.

**Pull request merged**